src/Routing/Router.php

```diff
@@ -594,6 +594,8 @@ public static function url($url = null, $full = false)
                 unset($url['_ssl']);
             }
 
+            $url = static::_applyUrlFilters($url);
```

Comment:
Yes there is a tiny likely hood of that happening though in current form this feature isn't very useful imho. Also I don't think many would be aware of url filters so i would rather change it now than say wait till 3.1